packages/react-components/react-components/src/Concepts/Icons/Icon.stories.mdx

```diff
@@ -1,7 +1,6 @@
 import { Meta } from '@storybook/addon-docs';
-import { Spinner } from '@fluentui/react-spinner';
 
-<Meta title="Concepts/Developer/Icons" />
+<Meta title="Concepts/Developer/Icons/Icons" />
```

Comment:
Two levels of Icons?  Sounds like Little Caesar's - Pizza Pizza.

Reply:
![SundayMonsterGIF](https://user-images.githubusercontent.com/1434956/186262731-e9402d9f-7a28-4d1c-8f20-b6acc4162013.gif)
ICONS
ICONS
ICONS!